src/pds/registrysweepers/utils/db/client.py

```diff
@@ -18,7 +18,14 @@ def get_opensearch_client_from_environment(verify_certs: bool = True) -> OpenSea
 
 
 def get_opensearch_client(endpoint_url: str, username: str, password: str, verify_certs: bool = True) -> OpenSearch:
-    scheme, host, port = endpoint_url.replace("://", ":", 1).split(":")
+    try:
+        scheme, host, port_str = endpoint_url.replace("://", ":", 1).split(":")
```

Comment:
Is this better than [urllib.parse](https://docs.python.org/3/library/urllib.parse.html)?